crates/ruff/resources/test/fixtures/flake8_logging/LOG007.py

```diff
@@ -0,0 +1,10 @@
+import logging
```

Comment:
Can you add a test case with `logger.exception`? (https://github.com/adamchainz/flake8-logging/blob/8cd23181647d4c68391af1db71588e8b3dc2bcfe/tests/test_flake8_logging.py#L783-L793)